go.sum

```diff
@@ -318,9 +316,8 @@ github.com/go-logr/logr v0.1.0/go.mod h1:ixOQHD9gLJUVQQ2ZOR7zLEifBX6tGkNJF4QyIY7
 github.com/go-logr/logr v0.2.0/go.mod h1:z6/tIYblkpsD+a4lm/fGIIU9mZ+XfAiaFtq7xTgseGU=
 github.com/go-logr/logr v0.4.0 h1:K7/B1jt6fIBQVd4Owv2MqGQClcgf0R266+7C/QjRcLc=
 github.com/go-logr/logr v0.4.0/go.mod h1:z6/tIYblkpsD+a4lm/fGIIU9mZ+XfAiaFtq7xTgseGU=
+github.com/go-logr/zapr v0.1.0 h1:h+WVe9j6HAA01niTJPA/kKH0i7e0rLZBCwauQFcRE54=
```

Comment:
Probably, slipped through?

Reply:
Hi, unfortunately not :   
```sh
❯ go mod graph | grep zapr
sigs.k8s.io/controller-runtime@v0.6.4 github.com/go-logr/zapr@v0.1.0
sigs.k8s.io/controller-runtime@v0.4.0 github.com/go-logr/zapr@v0.1.0
```

graph command shows what is dependent on the left and dependencies on the right. if package has dependencies, it is shown in the table.

Reply:
I see.